content/blog/posts/for-the-love-of-the-game.md

```diff
@@ -0,0 +1,109 @@
+
+**Jacob:**  Admittedly, I have less experience in open source than Phoenix, and I hope to learn more through my involvement with O3DE. My first hands-on experience with open source really started at the beginning of last year when we began seriously talking with another friend of ours about using the Open 3D Engine.
+
+About a decade ago, I had some experiences that opened my eyes to how damaging and potentially dangerous it could be for people to use proprietary software. It was at that time that I became very interested in open source because of its ability to be audited by anybody and contributed to by anybody with the know-how and willingness to do so. So, for me, O3DE was a nice change of pace when I heard about it.
+
+**How did you discover the Open 3D Engine? What was your path into the O3DE community?**
+
+**Phoenix:**  A couple years back, I tested out Amazon Lumberyard. I’d been keeping an eye on Lumberyard for a while when I heard that it was being open-sourced and evolving into Open 3D Engine (O3DE). That was pretty much how I found out about O3DE early on, pre-release.
+
+**Can you describe your early experiences with the Open 3D Engine?**
+
+**Phoenix:** In terms of onboarding, O3DE is very similar in some aspects, such as UI and function-wise, to both [Unreal Engine](https://www.unrealengine.com/en-US/) and [Unity](https://unity.com/). Any prior experience with either of those engines transfers over really well to O3DE. For example, I found it really easy to pick up O3DE having a background in Unreal Engine.
+
+And then, you said you found it more intuitive to pick it up from scratch than you did Unreal Engine, that things just felt more natural to you, right?
+
+**Jacob:**  Yeah. We’d been using the Unreal Engine for prototyping, and we sat down a few months ago to look at licensing and all of the different aspects of it. We ended up creating a chart of many of the major engines, including O3DE and another open-source engine called [Flax](https://flaxengine.com/), and scoring them across a few different categories. It turns out, O3DE came out on top for us by almost 10 points above any of the other engines.
+
+Onboarding in O3DE, on a scale of 1-5, was a five because we found that the [documentation](https://www.o3de.org/docs/) was great, better than anything else we’d experienced. When using other engines, I couldn’t find the answers I was looking for in the documentation, so I’d try to find the answers in a YouTube video or a forum that explained what we were looking for. In O3DE, everything is explained in the documentation. We can also hop into [Discord](http://discord.com/invite/o3de) and talk with the people who are developing the engine and can give us the answers we’re looking for on the spot, maybe that day or the next day. So, the onboarding for O3DE, and the community, have been amazing.
```

Comment:
Docs are internal, Change 
`[documentation](https://www.o3de.org/docs/) `
to
`[documentation](/docs/) `